core/src/env2/call.rs

```diff
@@ -59,11 +59,13 @@ impl<'a> From<&'a [u8]> for Selector {
 
 impl Selector {
     /// Returns the selector for the given name.
+    #[allow(clippy::should_implement_trait)]
     pub fn from_str(name: &str) -> Self {
         From::from(name.as_bytes())
     }
 
     /// Returns the underlying bytes of the selector.
+    #[allow(clippy::trivially_copy_pass_by_ref)]
     pub fn to_bytes(&self) -> [u8; 4] {
         self.bytes
     }
```

Comment:
We won't implement the associated traits here because traits cannot be `const fn` which is a major pita for the `Selector` type which is going to be used in `const fn` contexts.